activesupport/lib/active_support/values/time_zone.rb

```diff
@@ -347,14 +347,14 @@ def at(secs)
     # components are supplied, then the day of the month defaults to 1:
     #
     #   Time.zone.parse('Mar 2000') # => Wed, 01 Mar 2000 00:00:00 HST -10:00
-    def parse(str, now=now())
```

Comment:
I thought about that. But I wanted to push the least number of changes i.e. simply change the name of the variable. If a good number of people suggest that I remove the `()` from the original code, then I shall.